### PR TITLE
[ROADMAP] tightens communications

### PR DESCRIPTION
**Pull request by ItsVyzo**
# About the pull request
Tightens communications as per roadmap https://github.com/orgs/cmss13-devs/projects/6/views/1?pane=issue&itemId=23382941

# Explain why it's good for the game
Balance

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: LTNTS
balance: Limited communications for PO/DCC/CT/MST/CL/VC
fix: IO/VC now use their own encryption keys
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -993,6 +993,7 @@
 
 /obj/item/device/radio/headset/almayer/mcom/vc
 	name = "marine vehicle crew radio headset"
-	desc = "Used by USCM vehicle crew, features a non-standard brace. Channels are as follows: :v - marine command, :a - alpha squad, :b - bravo squad, :c - charlie squad, :d - delta squad, :n - engineering, :m - medbay, :u - requisitions, :j - JTAC, :t - intel."
+	desc = "Used by USCM vehicle crew, features a non-standard brace. Channels are as follows: :v - marine command, :n - engineering, :m - medbay, :u - requisitions, :j - JTAC, :t - intel."
+	initial_keys = list(/obj/item/device/encryptionkey/po)
```
Comment (morrowwolf) [inline code/game/objects/items/devices/radio/headset.dm:997]: They should be using their own key.

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by SarahJohnson489** [comment]
Incredibly dumb PR that ignores req communicating with squads for their supplies, CAS dragnetting surviving marines with recovered binos/flares from permas, and transport sending out cycle notifications. Ignores that the presence of the channels helps a severely isolated role feel involved with the happenings in the round instead of a muffled NPC in a slave role filling boxes or pressing the funny transport button every so often in a void. PR helps nothing and just shows more retarded decision-making seeking to make problems where there were none.

**Comment by ItsVyzo** [comment]
> Otherwise, I'd like to see how this effects things. I'm not sold one way or the other but let's see how it goes.

It will be an interesting change, however I think depending on how limited communication is it could either be good or bad. Also, with the new RTO changes, it might push people to use radiopacks more. Good portion of this really depends on marine competence.

**Review by morrowwolf — APPROVED** [review]
This along with radio backpacks being more distributed should allow for communications to be a bit tighter without issue.

**Pull request merged**